wrappers/python/tests/conftest.py

```diff
@@ -11,23 +11,30 @@ def starpu_simple():
 
 
 @pytest.fixture(scope="package")
-def numpy_fixed_seed():
-    # import here, only if used
+def numpy_rng():
     import numpy as np
     import random
 
     seed = 42
     np.random.seed(seed)
     random.seed(seed)
+    bits = np.random.MT19937()
+    rng = np.random.Generator(bits)
+    return rng
 
 
 @pytest.fixture(scope="package")
-def torch_fixed_seed(numpy_fixed_seed):
-    # import here, only if used
+def torch_rng():
     import torch
 
     seed = 42
+
     torch.manual_seed(seed)
     torch.cuda.manual_seed(seed)
     torch.backends.cudnn.enabled = False
     torch.backends.cudnn.deterministic = True
+
+    gen = torch.Generator()
+    gen.manual_seed(seed)
+
+    return gen
```

Comment:
Are you sure about `scope`? Result still depends on order in which tests are run.

It is up to you but is this `torch.Generator` used anywhere? What if `torch` is CPU-only like in GitHub CI? Is it needed only to check correctness of `nntile`'s routines?

Reply:
I use it only for seed. Rng creation only for compatibility 
If you have plans on this, you can do it in separate PR. I think it is out of scope wight now

Reply:
> If you have plans on this, you can do it in separate PR. I think it is out of scope wight now

This is exactly why I suggested to create rng in-place and use it without fixtures.

Reply:
Ok, thanks. I believe this ok too. You can fix it later